tests/async/test_async_model_methods.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase
+
+from .models import Book
+
+
+class AsyncModelOperationTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.s1 = Book.objects.create(
+            title="David Copperfield",
+        )
+        cls.s2 = Book.objects.create(
+            title="The Great Gatsby",
+        )
```

Comment:
Only one model instance is needed in the below tests, so let’s avoid creating two.

```suggestion
        cls.s1 = SimpleModel.objects.create(field=0)
```